Formula/fn.rb

```diff
@@ -0,0 +1,25 @@
+class Fn < Formula
+  desc "CLI tool for the fn project."
+  homepage "https://fnproject.github.io"
+  url "https://github.com/fnproject/cli/archive/0.3.82.tar.gz"
+  sha256 "c1e8aa750df890cb4a69d5b877a5562da4d9f6f65b3b36c65a649a33be1ecf5d"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/fnproject/cli"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install", "-v", "--force", "--skip-test"
+      system "go", "build", "-o", "#{bin}/fn"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/fn --version")
```

Comment:
We need a test that does more than matching version from the output.

Reply:
Noticed several others that do this so thought it was OK: https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker.rb#L45 . In any case, added more tests.